lib/tasks/major-update/ui.js

```diff
@@ -22,7 +22,7 @@ module.exports = function ui(ctx) {
             ctx.ui.log(`${logSymbols.success} Your theme is compatible.\n`);
 
             if (demoPost && demoPost.uuid) {
-                const demoLink = `${ctx.instance.config.get('url')}p/${demoPost.uuid}/`;
+                const demoLink = `${ctx.instance.config.get('url')}/p/${demoPost.uuid}/`;
```

Comment:
you probably also want to make sure you don't get a double trailing slash

```js
const demoLink = `${ctx.instance.config.get('url').replace(/\/$/,'')}/p/${demoPost.uuid}/`;
```

Reply:
Thanks! 😄